src/main/app/src/app/zaken/zaak-details-wijzigen/zaak-details-wijzigen.component.ts

```diff
@@ -302,13 +302,6 @@ export class CaseDetailsEditComponent implements OnInit, OnDestroy {
     const isSameGroup = zaak.groep?.id === this.zaak.groep?.id;
     if (isSameBehandelaar && isSameGroup) return EMPTY;
 
-    if (zaak.behandelaar?.id === this.loggedInUser.id) {
```

Comment:
Needed to allow backend to be called every time the user changes

Currently if user X is logged in and zaak is assigned to user Y frontend does not make a call to backend